src/hotspot/share/runtime/deoptimization.cpp

```diff
@@ -48,6 +48,7 @@
 #include "runtime/deoptimization.hpp"
 #include "runtime/frame.inline.hpp"
 #include "runtime/interfaceSupport.inline.hpp"
+#include "runtime/jniHandles.inline.hpp"
```

Comment:
In the original PR this include already exists but it is just moved around. I just added it in this PR since it didn't exist yet at all in jdk11.